docs/getting-started/01-overview.md

```diff
@@ -0,0 +1,81 @@
+
+- [Commerce mock](https://github.com/SAP-samples/xf-addons/tree/master/addons/commerce-mock-0.1.0) is to act as an example of an external monolithic application which we want to extend with Kyma. It is based on the [Varkes](https://github.com/kyma-incubator/varkes) project and is also available in the form of an addon. It will simulate how you can pair an external application with Kyma and expose the application's APIs and events. In these guides, you will use its `order.deliverysent.v1` event type to trigger both the microservice and the Function.
+
+## Steps
+
+These guides cover the following steps:
+
+1. [Create the `orders-service` Namespace](#getting-started-create-a-namespace) in which you will deploy all resources.
+2. [Create a microservice](#getting-started-deploy-a-microservice) by deploying `orders-service` on the cluster in the created Namespace.
+3. [Expose the microservice](#getting-started-expose-the-microservice) through the APIRule custom resource (CR) on HTTP endpoints. This way it will be available for other services outside the cluster so you can retrieve data from it and send sample orders to it. Since the microservice uses in-memory storage, when you send a sample order to its endpoint, the order details will be gone after you restart the microservice.
+4. [Add the Redis service](#getting-started-add-the-redis-service) as an addon.
+5. [Create the Redis service instance](#getting-started-create-a-service-instance-for-the-redis-service) (ServiceInstance CR) in the Namespace so you can bind it with the microservice and Function.
+6. [Bind the microservice to the Redis service](#getting-started-bind-the-redis-service-instance-to-the-microservice) by creating ServiceBinding and ServiceBindingUsage CRs. Send a sample order to its endpoint. Since it now uses the Redis storage, the order details will not be gone after you restart the microservice.
+7. [Connect Commerce mock](#getting-started-connect-an-external-application) as the external application.
+8. [Trigger the microservice](#getting-started-trigger-the-microservice-with-an-event) to react to the `order.deliverysent.v1` event from Commerce mock. Send the event, and then see if the microservice reacts to it and saves its details in the Redis database.
+9. [Create a Function](#getting-started-create-a-function) in the Namespace and repeat the microservice flow:
+- [Expose the Function](#getting-started-expose-the-function) through the APIRule CR on HTTP endpoints. This way it will be available for other services outside the cluster.
+- [Bind the Function to the Redis service](#getting-started-bind-a-redis-service-instance-to-the-function) by creating ServiceBinding and ServiceBindingUsage CRs.
+- [Trigger the Function](#getting-started-trigger-the-function-with-an-event) to react to the `order.deliverysent.v1` event from Commerce mock. Send the event, then and see if the Function reacts to it and saves its details in the Redis database.
```

Comment:
```suggestion
- [Trigger the Function](#getting-started-trigger-the-function-with-an-event) to react to the `order.deliverysent.v1` event from Commerce mock. Send the event, and then see if the Function reacts to it and saves its details in the Redis database.
```
I think we have an unintended slip-up here.
I'd personally go without the and, but if you'd rather add it, that's alright (applies also to line 73).

Reply:
Is that ok?
```
Send the event. Then, see if the microservice reacts to it and saves its details in the Redis database.
```